dev/Breadcrumb/Breadcrumb.xaml

```diff
@@ -17,7 +17,7 @@
             <Setter.Value>
                 <ControlTemplate TargetType="local:BreadcrumbItem">
 
-                    <Button x:Name="PART_BreadcrumbItemButton">
+                    <Button x:Name="PART_BreadcrumbItemButton" AutomationProperties.Name="BreadcrumbItemButton">
```

Comment:
I think this should be a localized resource.

Reply:
Yes, all AutomationProperties.Name values need to be localized, Which I believe means you will need to set this from code behind.  take a look at this for an example: https://github.com/microsoft/microsoft-ui-xaml/blob/a146dd3db9d6b78a46521099944b74934a9be650/dev/NumberBox/NumberBox.cpp#L142